src/node-red/index.njk

```diff
@@ -55,99 +49,115 @@ meta:
+                                </div>
+                                <div class="w-full flex flex-row gap-3 mx-auto md:m-0">
+                                    <h5 class="w-full md:m-0">
+                                        <div class="text-xl text-gray-600 text-center sm:text-left">{{ section.title }}</div>
+                                    </h5>
+                                </div>
+                            </div>
+                            <div>
+                                <p class="text-center sm:text-left font-light">{{ section.description | safe }}</p>
+                            </div>
                         </div>
-                    </h3>
-                    <p class="mb-0">
-                        <a href="https://dashboard.flowfuse.com/" target="_blank">Node-RED Dashboard</a> makes it easy to visualize your data in charts that integrate directly with actions and events that respond to the data.
-                    </p>
-                </div>
+                    </div>
+                {% endfor %}
+
             </div>
         </div>
     </div>
 </div>    
 
-<!--Customer Stories Banner-->
-<div class="w-full text-center px-6 pb-3 pt-8 text-lg flex-col flex-row items-center justify-center gap-1 hover:no-underline flex-wrap">
-    <h3>
-        <span class="hover:no-underline">
-            Node-RED use cases from our customer stories
-        </span>
-    </h3>
-    <ul class="mt-6 grid grid-cols-1 md:grid md:grid-cols-3 gap-y-4 pb-4 max-w-screen-lg m-auto">
+<!--Learning Resources-->
+<div class="w-full bg-gray-50 pb-20 pt-16">
+    <div class="max-w-screen-lg m-auto px-6">
+    <h2 class="max-md:text-center">
+        Node-RED <span class="text-indigo-600">Learning Resources</span>
+    </h2>
+<div class="max-sm:text-center max-w-md sm:max-w-none mx-auto sm:grid justify-center items-center mt-10" style="grid-template-columns: 45% auto;">
+    <div class="max-h-[300px] md:max-h-[272px] ff-image-cover scale ff-image-rounded w-full h-full mb-4 sm:mb-0">
+        {% image "../images/learning_nr.png", "Node-RED pseudo UI", [406] %}
+    </div>
+        <div class="content-center justify-center sm:pl-8 flex-1">
+            <p class="md:mt-0">
+                Jumpstart your Node-RED journey with FlowFuse’s comprehensive learning resources, including tutorials, case studies, and community forums. Whether you’re new to Node-RED or looking to refine your skills, our resources are designed to accelerate your learning curve.
+            </p>
+            <p class="text-blue-700 mb-10 mt-7">
+                <a href="/node-red/core-nodes/">Core Nodes</a>
+                <span class="px-3">|</span>
+                <a href="/blog/2024/05/understanding-node-flow-global-environment-variables-in-node-red/">Variables</a>
+                <span class="px-3">|</span>
+                <a href="http://localhost:8080/blog/2023/04/3-quick-node-red-tips-6/">Quick Tips</a>
```

Comment:
```suggestion
                <a href="/blog/2023/04/3-quick-node-red-tips-6/">Quick Tips</a>
```

Reply:
Is there a quick way to have a page/link that has all the quick tip blogs?

Reply:
This is the way at the moment, there's a list with all the links to the other articles at the end of each one. We'll take care of it on the next iteration.